sos/report/plugins/docker.py

```diff
@@ -82,7 +82,7 @@ def setup(self):
         for container in containers:
             self.add_cmd_output("docker inspect %s" % container,
                                 subdir='containers')
-            if self.get_option('logs'):
+            if self.get_option('logs') or self.get_option("all_logs"):
```

Comment:
I'm unsure about this one. I see the rationale for it, but `all-logs` has traditionally been more related to size considerations than gating command output.

Reply:
I've left this the same, but if it needs changing let me know.